packages/mui-base/src/Button/Button.types.ts

```diff
@@ -30,6 +30,11 @@ export interface ButtonOwnProps extends Omit<UseButtonParameters, 'rootRef'> {
    * @default {}
    */
   slots?: ButtonSlots;
+  /**
+   * The HTML element, e.g.'button', 'a' etc
+   * @default 'button'
+   */
```

Comment:
Let's make it more explicit. Also, I remember Sam suggesting limiting the use of Latin abbreviations.

```suggestion
  /**
   * The HTML element that is ultimately rendered, for example 'button', 'a', etc.
   * @default 'button'
   */
```